docs/usage/config-overview.md

```diff
@@ -0,0 +1,299 @@
+
+## Shared Presets
+
+### Overview
+
+The concept of shared configuration is covered in detail on the [Presets](./key-concepts/presets.md) page, so please read that first.
+
+### Use of Presets in Global config
+
+Presets should be used cautiously in Global config as they often lead to misunderstandings.
+
+#### globalExtends
+
+Sometimes you may not wish to put all settings within the Global config itself and instead commit it to a repository which is then referenced from the Global config.
+In such cases, use `globalExtends` instead of `extends` so that it is resolved immediately and used as part of Global config.
+
+#### extends
+
+If you use `extends` within Global config then it's important to note that these are _not_ resolved/expanded during Global config processing and instead are passed through unresolved to be part of Repository config.
+Passing `extends` through to be part of Repository config has two major consequences:
+
+- It allows repository users to be able to use `ignorePresets` to ignore all or part of the `extends` presets, and
+- Presets defined within `extends` in Global config will take _higher_ precedence that "regular" Global config, because it's resolved later
+
+### Using a centralized config
+
+Using "centralized" configs through Renovate presets is important in order to be able to:
+
+- Save time by not repeating yourself in every repo with the same config, and
+- Being able to change settings across an entire Organization or groups of repositories in one place
+
+Once you've created a centralized preset config, there are multiple ways you can pass it through to repositories:
+
+- Defining it in Global config (either `globalExtends` or `extends`)
+- Using it as your Inherited config, or referencing it from Inherited config using `extends`
+- Ensuring it's referenced in Onboarding config so that it's committed as part of the Repository config
+
+The above possibilities go from least to most transparent when it comes to end users.
+
+Global config may be invisible to developers without log access, meaning they could be confused by any settings you apply - via presets or directly - within Global config.
+For example the developers wonder why Renovate is behaving differently to its documented default behavior and may even think it's a bug.
+
+Inherited config is visible to developers (it's within a repository they can see) although it's _implicitly_ applied so without log access and if they're not aware to look for an Inherited config repository then they may again be a little confused as to why default behavior has changed.
+
+Extending presets through Repository config is the most explicit and obvious way to use presets from an end user point of view.
+If they wonder why behavior is a certain way, they can look at the Repository config and trace through all presets it references.
```

Comment:
This section is hard to understand so I've stayed away from it.

I would prefer to have examples how to _avoid_ confusion. Right now it seems to explain how to create confusion.

Reply:
I've tried to improve this text with an edit